persistence/scaladsl/src/main/scala/com/lightbend/lagom/internal/scaladsl/persistence/PersistentEntityActor.scala

```diff
@@ -142,11 +141,8 @@ private[lagom] class PersistentEntityActor(
   }
 
   private def unhandledCommand: PartialFunction[(C, entity.CommandContext[Any], S), entity.Persist] = {
-    case (cmd, _, _) =>
-      // not using akka.actor.Status.Failure because it is using Java serialization
-      sender() ! PersistentEntity.UnhandledCommandException(
-        s"Unhandled command [${cmd.getClass.getName}] in [${entity.getClass.getName}] with id [${entityId}]"
-      )
+    case (cmd, ctx, state) =>
+      entity.onUnhandledCommand(cmd, ctx.asInstanceOf[entity.ReadOnlyCommandContext[Nothing]], state)
```

Comment:
could users have been relying on something sent back to sender?

Reply:
the default implementation of entity.onUnhandledCommand is exactly the same, although it is via ctx.invalidCommand, which will send it to sender.